bin/build-acorn.js

```diff
@@ -3,14 +3,30 @@ var stream = require("stream")
 
 var browserify = require("browserify")
 var babelify = require("babelify")
+var derequire = require("derequire")
 
 process.chdir(path.resolve(__dirname, ".."))
 
+function derequireStream() {
```

Comment:
Oh, I meant you can just use ready-to-use [browserify plugin](https://github.com/rse/browserify-derequire) so that it would really take just 3 lines diff.